markdown/ch1.markdown

```diff
@@ -57,59 +57,62 @@ def rpkm(counts, lengths):
 
 To access this information, the DNA is *transcribed* into a sister molecule called *messenger ribonucleic acid*, or mRNA.
 Finally, this mRNA is *translated* into proteins, the workhorses of the cell.
 A section of DNA that encodes the information to make a protein (via mRNA) is called a gene.
 
 The amount of mRNA produced from a given gene is called the *expression* of that gene.
 Although we would ideally like to measure protein levels, this is a much harder task than measuring mRNA.
-Fortunately, expression levels of an mRNA and levels of its corresponding protein are usually correlated ([Maier, Güell, and Serrano, 2009](http://www.sciencedirect.com/science/article/pii/S0014579309008126)).
+Fortunately, expression levels of an mRNA and levels of its corresponding protein are usually correlated.[^Maier]
+
+[^Maier]: Tobias Maier, Marc Güell, and Luis Serrano. ["Correlation of mRNA and protein in complex biological samples"](http://www.sciencedirect.com/science/article/pii/S0014579309008126), FEBS Letters 583, no. 204 (2009).
+
 Therefore, we usually measure mRNA levels and base our analyses on that.
 As you will see below, it often doesn't matter, because we are using mRNA levels for their power to predict biological outcomes, rather than to make specific statements about proteins.
 
 <img src="../figures/central_dogma.png"/>
-<!-- caption text="Central Dogma of Molecular Biology" -->
+<!-- caption text="Central dogma of molecular biology" -->
 
 It's important to note that the DNA in every cell of your body is identical.
 Thus, the differences between cells arise from *differential expression* of
 that DNA into RNA: in different cells, different parts of the DNA are processed
-into downstream molecules. Similarly, as we shall see in this chapter and the
+into downstream molecules. Similarly, as we will see in this chapter and the
 next, differential expression can distinguish different kinds of cancer.
 
 <img src="../figures/differential_gene_expression.png"/>
 <!-- caption text="Gene expression" -->
 
 The state-of-the-art technology to measure mRNA is RNA sequencing (RNAseq).
-RNA is extracted from a tissue sample, for example from a biopsy from a patient, *reverse transcribed* back into DNA (which is more stable), and then read out using chemically modified bases that glow when they are incorporated into the DNA sequence.
-Currently, high-throughput sequencing machines can only read short fragments (approximately 100 bases is common). These short sequences are called “reads”.
+RNA is extracted from a tissue sample (e.g., from a biopsy from a patient), *reverse transcribed* back into DNA (which is more stable), and then read out using chemically modified bases that glow when they are incorporated into the DNA sequence.
+Currently, high-throughput sequencing machines can only read short fragments (approximately 100 bases is common). These short sequences are called “reads.”
```

Comment:
Same as I saw before; is this punctuation correct?

Reply:
Sadly, yes. As a computer scientist, I hate it, but it's what English style guides all recommend: punctuation inside the quote marks.

Reply:
That's just wrong!!

Reply:
In terms of grammar and punctuation - that depends on America vs Britain English language. For example:

http://www.thepunctuationguide.com/british-versus-american-style.html

http://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks

https://www.theguardian.com/media/mind-your-language/2011/may/19/mind-your-language-punctuation-quotations

Reply:
Thanks @i-s-h-i-m! That was both enlightening and confusing. =D I had to go pull out books by different authors to verify some of the claims, and they checked out! I had never consciously noticed British authors using single quotes by default, but sure enough, there it was!

My new policy is: do as I please and then let the editor sort it out. =P

Reply:
Well, I'd be!